lib/liquid/combinators/tags/if.ex

```diff
@@ -28,8 +28,22 @@ defmodule Liquid.Combinators.Tags.If do
     |> optional(parsec(:__parse__))
   end
 
+  @type unless_tag :: [unless: [If.t()]]
```

Comment:
`unless` is not `[unless: [if: [conditions: ...]]` is `[unless: [conditions: ...]]` you must create an `if_body` type and reuse it for `if` and `unless`